pkgs/applications/misc/zathura/core/default.nix

```
@@ -4,6 +4,9 @@
 , sqlite, glib, texlive, libintl, libseccomp
 , file, librsvg
 , gtk-mac-integration
+, syncTexSupport ? stdenv.buildPlatform.canExecute stdenv.hostPlatform  # does not cross compile
+, manpageSupport ? stdenv.buildPlatform.canExecute stdenv.hostPlatform
```

```suggestion
, manpageSupport ? true
```
Since the manpages also cross-compile (now that we take sphinx from buildPackages), there isn't really a reason not to include them in the cross build by default, is there?